src/override/vs/base/common/marked/marked.ts

```diff
@@ -0,0 +1,4 @@
+import marked from 'vscode-marked'
```

Comment:
Can you explain what this does / what "feat: replace inlined marked by dependency" really means? Thanks.

Reply:
Sorry if it not clear.

What was done here is very similar to what is already done for semver. For some reasons, VSCode inlines marked and semver in their codebase. It's an issue for us because the inlined version is in CommonJS and the rollup treeshaking breaks it (because of the use of IIFEs).

Any file in the `override` directory will be used instead of the corresponding VSCode file, I use it to use the official semver package (with the corresponding old version) instead of the file inlined in VSCode

I don't find any documentation on why they do it, [I've ask but I don't really expect any answer soon](https://stackoverflow.com/questions/77872402/why-is-semver-and-marked-inlined-in-vscode)

Reply:
Thank you. Did you ask that at stackoverflow because there was no answer in the VSCode repo?

Reply:
I've ask to stackoverflow because the issue workflow on the VSCode repo redirects to it when you want to ask questions

Reply:
Thanks, I didn't know that